src/MSBuild/Resources/Strings.resx

```diff
@@ -1184,6 +1184,13 @@
   <data name="PossiblyOmittedMaxCPUSwitch" xml:space="preserve">
     <value>Building the projects in this solution one at a time. To enable parallel build, please add the "-m" switch.</value>
   </data>
+  <data name="CachePluginProjectFinished_OutputPath" xml:space="preserve">
+    <value> -> {0}</value>
```

Comment:
This was suggested by @AndyGerlicher. If preferred to remove, I can just use the existing string, but I think as a short term solution, we want to be able to demonstrate that projects were built with the project cache plugin enabled rather than suppressing that output. When TL is enabled and PCP is disabled, the output comes through, so we want to keep that functionality.